### PR TITLE
infra chart removing

### PR DESCRIPTION
**Pull request by almerico**
fixes ACS-289

### CODE REVIEW

File: helm/alfresco-content-services/requirements.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ dependencies:
   version: 0.9.3
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: postgresql.enabled
-- name: alfresco-infrastructure
-  version: 4.1.1
-  condition: alfresco-content-services.alfresco-infrastructure.enabled
+- name: activemq
+  version: ^1.1.1
```
Comment (mteodori) [inline helm/alfresco-content-services/requirements.yaml:13]: can we put a fixed version 1.1.1

File: helm/alfresco-content-services/templates/config-imagemagick.yaml [hunk 2]
```diff
@@ -12,10 +12,10 @@ data:
   {{ $key }}: {{ $val | quote }}
   {{- end }}
   {{- end }}
-  {{- if index .Values "alfresco-infrastructure" "activemq" "enabled" }}
+  {{- if index .Values "activemq" "enabled" }}
```
Comment (mteodori) [inline helm/alfresco-content-services/templates/config-imagemagick.yaml:15]: we can go with the simpler .Values.activemq.enabled as we don't have the dash anymore

Comment (almerico) [reply helm/alfresco-content-services/templates/config-imagemagick.yaml]: Done

File: helm/alfresco-content-services/requirements.yaml (hunk 1 above)
Comment (almerico) [reply helm/alfresco-content-services/requirements.yaml]: done

**Comment by almerico** [comment]
https://bamboo.alfresco.com/bamboo/browse/PLAT-ACSHELM325-ADT-16 passed

**Pull request merged**